src/main/scala/com/typesafe/sbt/packager/docker/DockerPlugin.scala

```diff
@@ -615,7 +632,11 @@ object DockerPlugin extends AutoPlugin {
     log.debug("Executing Native " + buildCommand.mkString(" "))
     log.debug("Working directory " + context.toString)
 
-    val ret = sys.process.Process(buildCommand, context) ! publishLocalLogger(log)
+    val logger = publishLocalLogger(log)
+    val ret = sys.process.Process(buildCommand, context) ! sys.env
+      .get("DOCKER_BUILDKIT")
+      .map(_ => publishLocalBuildkitLogger(log))
+      .getOrElse(logger)
```

Comment:
maybe put the decision logic here. I always find the `!` syntax of the `sys.process.Process` already hard  to read and with additional logic behind, it gets even harder.

```suggestion
    val logger = sys.env.get("DOCKER_BUILDKIT")
       .exists(_ == "1")
       .map(_ => publishLocalBuildkitLogger(log))
       .getOrElse(publishLocalLogger(log))
     val ret = sys.process.Process(buildCommand, context) ! logger
```

Reply:
That would change the logger for the other cmd running `docker images prune...`. Not sure this is a good idea ? Or shall I create another variable ?

Reply:
I would keep it consistent. One build kit logger and one if no buildkit is being in use.

So yeah. I'm fine with changing the logger for the prune command if buildkit is enabled

Reply:
It will potentially output in INFO actual errors from the pruning though. But if you're fine with it, I made the change.